sources/metricdog/src/main_test.rs

```diff
@@ -123,8 +123,8 @@ fn opt_out() {
 #[test]
 /// assert that send-boot-success exits without error even when there is no HTTP server
 fn send_boot_success_no_server() {
-    let port = 0;
-    let tempdir = create_test_files(port, &[], true);
+    let metrics_url = "http://localhost:0/metrics";
```

Comment:
This localhost:0 is fine because this test is ensuring a failed request does not cause the program to exit non-zero.